packages/domain-search/src/page/test/results.tsx

```diff
@@ -176,6 +176,34 @@ describe( 'ResultsPage', () => {
 	} );
 
 	describe( 'FQDN suggestion', () => {
+		it( 'shows FQDN suggestion if it is available, even if it is not in the suggestions list', async () => {
```

Comment:
I'm a bit concerned about this test title. Why would it not be in the suggestions list? Do you get it? And I also think it's more about `allowedTlds` rather than `deemphasizedTlds` because it's about filtering, not TLD categorization.

Reply:
I've updated the test title to `adds FQDN suggestion to the suggestions list if the availability query is successful and it is available`, I think that makes more sense and explicits that the FQDN will be added to the suggestion list